CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Implementation of the RabbitMQ EventBus
 - DeadLetter APIs and memory implementation for storing events that failed delivery
 - RecipientRewriteTable Aliases and associated WebAdmin routes
+- Dead letter reDeliver routes on top of WebAdmin
```

Comment:
Is it DeadLetter or Dead letter?
And Why it's not EventBus Dead Letter? (some context would probably help the reader)